.github/workflows/update.yml

```diff
@@ -20,24 +20,6 @@ jobs:
       - name: Turnstyle
         uses: softprops/turnstyle@v1
 
-      - name: Set known_hosts
-        id: know-hosts
-        run: |
-          echo "::set-output name=BJ1::$(ssh-keyscan -H ${{ secrets.BJ_1_HOST }})"
-          echo "::set-output name=BJ2::$(ssh-keyscan -H ${{ secrets.BJ_2_HOST }})"
-          echo "::set-output name=LA1::$(ssh-keyscan -H ${{ secrets.LA_1_HOST }})"
-          echo "::set-output name=LA2::$(ssh-keyscan -H ${{ secrets.LA_2_HOST }})"
-
-      - name: Install SSH Key
-        uses: shimataro/ssh-key-action@v2
-        with:
-          key: ${{ secrets.SSH_KEY }}
```

Comment:
It seems that if this step is removed, we won't be able to sync files to the server.

Reply:
We still need to install ssh key to each git action, but we may not need set host to`~/.ssh/known_hosts` since we can use `-o StrictHostKeyChecking=no` to skip the `known_host` check .